src/main/java/org/jqassistant/contrib/plugin/php/scanner/parser/PHPFunctionParser.java

```diff
@@ -61,9 +62,39 @@ public PHPFunctionParser (Helper helper, PHPNamespaceDescriptor phpNamespace, Ma
     public PHPFunctionDescriptor parse(ParseTree tree){
         parseTree(tree, 1);
         
+        phpFunction.setSignature(getSignature(phpFunction));
+        
         return phpFunction;
     }
     
+    /**
+     * get Signature of function
+     * @param phpFunction
+     * @return String
+     */
+    protected String getSignature(PHPFunctionDescriptor phpFunction){
```

Comment:
`get` is a difficult term to use for anything but POJO gets. Please find a better one.